modules/ROOT/pages/relnotes-java-sdk.adoc

```diff
@@ -5,7 +5,7 @@
 
 At least Java 7 is required for current releases; all newer, vendor supported Java versions are supported (and encouraged). 
 See the xref:compatibility-versions-features.adoc[Compatibility] section for details.
-Java 8 or later is recommended since it brings performance enhancements and lambda expressions which are very convenient when using the asynchronous API.
+Java 11 is recommended which has various enhancements like lambda local variable type inference, profiling tools and updated security features. 
```

Comment:
I like recommending this, but I think we need to be clear that it's 2.7.2 and later.  I know there aren't likely issues with earlier versions, but we didn't test with Java 11 until 2.7.2.

Maybe just add "(starting with 2.7.2)" after "is recommended"?

I know it's annoying, but I kinda want to be able to be clear that say SDK 2.4 isn't supported on Java 11.

Reply:
Sure, fixed.